### ISSUE TITLE
build error on centos:6

### ISSUE THREAD

**Issue by NiuBlibing** [daschl/grok#21]
When build without clang, it throw the error:
```
error: failed to run custom build command for `onig_sys v69.7.1`

Caused by:
  process didn't exit successfully: `/tmp/deps/vector/vectordotdev-vector-da60b55/target/release/build/onig_sys-424fbb3a592c2f1a/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.56.0/src/lib.rs:1922:31
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Could remove the dependency of libclang by  adding `onig = { version = "6", default-features = false }`?

**Comment by daschl** [comment]
Fixed on main, will be part of 2.0.0. thanks for reporting!

**Comment by daschl** [comment]
2.0.0 is released, want to try it out?
